### PR TITLE
DAM-7047 Analytics UI - part 1 

### PR DESCRIPTION
**Pull request by zlepper**

### CODE REVIEW

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 1]
```diff
@@ -2606,3 +2606,373 @@ resource configservice_label rabbit_details_messages_purge_messages_purge_dialog
...
+  ]
+}
+
+resource configservice_label administration_tools_dashboard_tool_generate_member_logins {
+  key = 'ADMINISTRATION_TOOLS_DASHBOARD_TOOL_GENERATE_MEMBER_LOGINS'
+  group = 'administration-tools - dashboard-tool
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Generate member logins'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl:2744]:
```suggestion
      default_translation = 'Generate user logins'
```

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 2]
```diff
@@ -2606,3 +2606,373 @@ resource configservice_label rabbit_details_messages_purge_messages_purge_dialog
...
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Lav bruger logins'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl:2748]:
```suggestion
      default_translation = 'Lav bruger-logins'
```

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 3]
```diff
@@ -2606,3 +2606,373 @@ resource configservice_label rabbit_details_messages_purge_messages_purge_dialog
...
+  key = 'ADMINISTRATION_TOOLS_DASHBOARD_TOOL_GENERATE_ASSET_DOWNLOADS'
+  group = 'administration-tools - dashboard-tool
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Generate asset downloads'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Lav asset downloads'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl:2764]:
```suggestion
      default_translation = 'Lav asset-downloads'
```

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 4]
```diff
@@ -2606,3 +2606,373 @@ resource configservice_label rabbit_details_messages_purge_messages_purge_dialog
...
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_BLOCK_BUILDER_TIME_BUCKET_SIZE'
+  group = 'administration-tools - analytics-block-builder
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Time bucket size'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Tidsspandsstørrelse'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl:2828]:
```suggestion
      default_translation = 'Tids-spands-størrelse'
```

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 5]
```diff
@@ -2606,3 +2606,373 @@ resource configservice_label rabbit_details_messages_purge_messages_purge_dialog
...
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_BLOCK_BUILDER_START_DATE'
+  group = 'administration-tools - analytics-block-builder
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Start date'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Start dato'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl:2860]:
```suggestion
      default_translation = 'Startdato'
```

**Review by Poiar — CHANGES_REQUESTED** [review]

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl [hunk 6]
```diff
@@ -2606,3 +2606,373 @@ resource configservice_label rabbit_details_messages_purge_messages_purge_dialog
...
+  key = 'ADMINISTRATION_TOOLS_ANALYTICS_BLOCK_BUILDER_TIME_BUCKET_SIZE'
+  group = 'administration-tools - analytics-block-builder
+  product_id = resource.configservice_product.media_manager_5.id
+  default_label_values = [
+    {
+      default_translation = 'Time bucket size'
+      language_id = data.language.english.id
+    },
+    {
+      default_translation = 'Tids-spands-størrelse'
```
Comment (Poiar) [inline MM/5.9.0/config/media_manager_5/labels/Administration.dcl:2828]:
```suggestion
      default_translation = 'Tid-spands-størrelse'
```

**Review by Poiar — APPROVED** [review]

File: MM/5.9.0/config/media_manager_5/labels/Administration.dcl (hunk 6 above)
Comment (Poiar) [reply MM/5.9.0/config/media_manager_5/labels/Administration.dcl:2828]: I was unsure of which one was most correct. So, I approved this. You may choose to add this suggestion.

Comment (zlepper) [reply MM/5.9.0/config/media_manager_5/labels/Administration.dcl:2828]: Honestly, both kinda sucks, i have no idea what you would even call this thing in danish :D

**Pull request merged**